be/src/service/doris_main.cpp

```diff
@@ -60,7 +60,7 @@
 #include "common/signal_handler.h"
 #include "common/status.h"
 #include "io/cache/block/block_file_cache_factory.h"
-#include "io/fs/s3_file_write_bufferpool.h"
```

Comment:
Why need to include this?

Reply:
It was the previous implementation for eager buffer pool memory allocation. It's been removed now